lang-java-reach-soot/src/main/java/com/sap/psr/vulas/cg/soot/CustomEntryPointCreator.java

```diff
@@ -271,7 +274,7 @@ private SootMethod generateMethodImplementation(SootMethod methodToImplement,
         }
 
         JNopStmt startStmt = new JNopStmt();
-        JNopStmt endStmt = new JNopStmt();
+        //JNopStmt endStmt = new JNopStmt();
```

Comment:
This has been authored by @anddann, not sure the stmt can be commented out.

Reply:
The change looks nice!